src/test/java/com/example/project/dto/EmployeeData.java

```diff
@@ -0,0 +1,20 @@
+package com.example.project.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
this does not seem right, if we use gson, annotations should also come from it, probably we need `@SerializedName`